### ISSUE TITLE
Censor some warnings?

### ISSUE THREAD

**Issue by Thimoteus** [FrigoEU/psc-ide-vim#16]
I have a project that uses psa to censor some warnings that I don't care about, but psc-ide-vim will happily report them to me. Is there any way to censor them?

**Comment by FrigoEU** [comment]
Nope, not at the moment, but seems the other editor plugins are doing it so I guess I can't stay behind much longer :D.
